modules/akka-http/build.sbt

```diff
@@ -0,0 +1,16 @@
+name := "pureconfig-akka-http"
+
+crossScalaVersions ~= { _.filterNot(_.startsWith("2.11")) }
+
+libraryDependencies ++= Seq(
+  "com.typesafe.akka" %% "akka-actor" % "2.5.31",
```

Comment:
Why is akka-actor required here?

Reply:
It's a required dependency of akka-http.